Code/Yospace/Source/YospaceServerSideAdInsertionConfiguration.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  YospaceServerSideAdInsertionConfiguration.swift
+//
+//  Created by Raffi on 29/05/2024.
+//  Copyright © 2024 THEOplayer. All rights reserved.
+//
+
+import THEOplayerSDK
+
+/**
+ The type of the Yospace stream, represented by a value from the following list:
+    - live: The stream is a live stream.
+    - livepause: The stream is a live stream with a large DVR window.
+    - vod: The stream is a video-on-demand stream.
+ */
+@objc(THEOplayerYospaceStreamType)
+public enum YospaceStreamType: Int {
+    case live
+    case livepause
+    case vod
+}
+
+/** Represents a configuration for server-side ad insertion with the Yospace integration.*/
+@objc(THEOplayerYospaceServerSideAdInsertionConfiguration)
+public class YospaceServerSideAdInsertionConfiguration: NSObject, THEOplayerSDK.ServerSideAdInsertionConfiguration {
+    /** The identifier for the custom integration.*/
+    @objc public let integration: THEOplayerSDK.SSAIIntegrationId = .CustomSSAIIntegrationID
```

Comment:
You should also set `customIntegration = "yospace"` 😉

Maybe we should also expose a `CustomSsaiConfiguration` abstract class in the SDK itself, to make it easier for customers to create their custom configuration class? See [CustomSsaiDescription](https://www.theoplayer.com/docs/theoplayer/v7/api-reference/android/com/theoplayer/android/api/source/ssai/CustomSsaiDescription.html) in Android.

Reply:
Done. Instead of an abstract class I made a protocol (interface) with default implementation of `customIntegration`.